compiler/Bond/Template/CustomMapping.hs

```diff
@@ -30,11 +31,18 @@ data NamespaceMapping = NamespaceMapping
     , toNamespace :: QualifiedName
     }
 
+
```

Comment:
Yes of cause, I mentioned that I can use  simple `Parser` type-synonym only after this change.